internal/pod_models.go

```diff
@@ -859,15 +868,37 @@ func GetBackupDeployment(backup *fdbv1beta2.FoundationDBBackup) (*appsv1.Deploym
 	}
 
 	if len(backup.Spec.MainContainer.ImageConfigs) == 0 {
-		backup.Spec.MainContainer.ImageConfigs = []fdbv1beta2.ImageConfig{
-			{BaseImage: "foundationdb/foundationdb"},
+		if pointer.BoolDeref(backup.Spec.UseUnifiedImage, false) {
+			backup.Spec.MainContainer.ImageConfigs = []fdbv1beta2.ImageConfig{
+				{
+					BaseImage: "foundationdb/foundationdb-kubernetes",
+				},
+			}
+		} else {
+			backup.Spec.MainContainer.ImageConfigs = []fdbv1beta2.ImageConfig{
+				{
+					BaseImage: "foundationdb/foundationdb",
+				},
+			}
+		}
+	}
+
+	if len(backup.Spec.SidecarContainer.ImageConfigs) == 0 {
+		if !pointer.BoolDeref(backup.Spec.UseUnifiedImage, false) {
```

Comment:
if it's early enough, would it be helpful to make `false` a constant for all these similar checks that we can flip quickly later? e.g. `UseUnifiedImageDefault`? 
If it's not early, grep is probably easier haha

Reply:
We can move that into a method on the backup struct.